js/notify.js

```diff
@@ -1,30 +1,23 @@
 /**
  * User notification module
  */
-define([
-    'jquery',
-    'logger',
-    'config',
-    'templates/notification.hbs'
-], function($, Log, config, message_template) {
-    'use strict';
+import $ from 'jquery';
+import log from 'logger';
+import config from 'config';
+import message_template from 'templates/notification.hbs';
 
-    function show_message(message, type, container) {
-        container = container || config.notify_in;
-        $(container).prepend(message_template({
-            level: type == 'error' ? 'danger' : type,
-            message: message
-        }));
-        Log.debug('Notify:', {type: type, message: message, container: container});
-    }
+function show_message(message, type, container) {
+    container = container || config.notify_in;
+    $(container).prepend(message_template({
```

Comment:
I chose not to use insertBefore because when the element is empty is throws an error and does not insert the element. Plus msg is a string, not a DOM element.
And then I found out this `insertAdjacentHTML` method: https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML